Automattic/newspack-theme
# Remove previous/next navigation and associated styles

### All Submissions:


### Changes proposed in this Pull Request:

This PR removes the previous/next post navigation from the bottom of the single post template (both default and article feature). 

Links that were removed:

![image](https://user-images.githubusercontent.com/177561/62151855-2705e300-b2b6-11e9-8aaf-b591b13c7bab.png)

### How to test the changes in this Pull Request:

1. Apply the PR and run `npm run build`.
2. Check a single article using the default template; confirm there is no previous/next article links at the bottom, and no errors.
3. Check a single article using the Feature template; confirm there is no previous/next article links at the bottom, and no errors.
4. Confirm in both cases there's no spacing weirdness, or anything else like that, introduced.

### Other information:
